test/benchmarks/src/main/scala/scala/collection/immutable/ArraySeqBenchmark.scala

```diff
@@ -21,6 +19,7 @@ class ArraySeqBenchmark {
   var size: Int = _
   var integersS: ArraySeq[Int] = _
   var stringsS: ArraySeq[String] = _
+  val newS = Array("a", "b", "c", "d", "e", "f")
```

Comment:
I assume there's a good reason to initialise these in `@Setup`.

Reply:
I can't say I fully understand why that's needed, but I will move the initialisation to @Setup.